README.md

```diff
@@ -111,6 +119,20 @@ If you're using a Mac, you can open (double-click) the above file via Finder and
 
 Test coverage thresholds are defined in the `nyc` stanza of `package.json` and are currently all set to 50 i.e. 50% coverage. Feel free to edit these as per project requirements.
 
+## Usage
+
+_The following instructions assume you are running the app in vagrant, hence the urls referred to have [http://dev.chs-dev.internal:4904](http://dev.chs-dev.internal:4904) as their base. If you are running it elsewhere substitute in the appropriate base url and use the same extensions as those given in the instructions below._
+
+Once the app is up and running you should be able to view of all applications currently saved in the database at [http://dev.chs-dev.internal:4904/manage-applications](http://dev.chs-dev.internal:4904/manage-applications).
+
+From there you should be able to use the _Create an application_ button navigate to [http://dev.chs-dev.internal:4904/add](http://dev.chs-dev.internal:4904/add) where you will be able to create the details for, and submit, a new application. Applications come in three varieties; _live_, _test_ and _future_ which should correspond to different collections in the database (see the [developer hub api](https://github.com/companieshouse/applications.api.identity.ch.gov.uk) for more details on this).

```

Comment:
I'd suggest that we say they correspond to the different ubic services as users of this app theoretically aren't concerned with the database, but that might not be a very important distinction.

Reply:
I've made some changes to this section to try and clarify that distinction